### ISSUE TITLE
Releasing new gem version

### ISSUE THREAD

**Issue by Frexuz** [contentful/contentful_model#33]
Seems that master is ahead of the current gem.
It even has a fix for a specific issue I had (namespacing)

Would be nice to have it on rubygems :)

Update:
I tried getting the gem straight from github/master, but ran into #31 :(

**Comment by dlitvakb** [comment]
Hey @Frexuz,

#31 is now merged. I will wait on publishing a version onto RubyGems until we have a stable release, there might be bugs from the latests features I've added, and don't want to have people facing them unadvertedly.

Feel free to try using master, any bug report is more than welcome.

Cheers

**Comment by Frexuz** [comment]
Great :) Thanks for the merge!

**Comment by dlitvakb** [comment]
Hey @Frexuz,

I've just merged a new https://github.com/contentful/contentful-management.rb which includes some more serialization options for dealing with creating entries with relationships using this gem.

I'll wait for some usage feedback before releasing, but we're close to a new release.

Cheers
